### DISCUSSION TITLE
How long does it usually take before a cask get's a updated version?

### DISCUSSION THREAD

**Discussion by eskility** [Homebrew/discussions#4903] [Casks] [answered]
### Output of `brew config`

```shell
na
```

### Output of `brew doctor`

```shell
na
```

### Description of issue

Take the new .net 8.0 sdk as an example. How long will it take before I can do a brew upgrade --cask and upgrade to 8.0?

### Relevant casks

upgrade --cask

**Comment by SMillerDev** [comment, marked as answer]
Until someone does https://docs.brew.sh/Updating-Software-in-Homebrew, and that someone could be you.

And then until a volunteer has some time to review it
